easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.9.1-gompi-2019a.eb

```diff
@@ -0,0 +1,36 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+easyblock = 'CMakeMake'
+
+name = 'mpifileutils'
+version = '0.9.1'
+
+homepage = 'https://hpc.github.io/%(name)s/'
+description = """
+ MPI-Based File Utilities For Distributed Systems
+"""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+
+source_urls = ['https://github.com/hpc/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['15a22450f86b15e7dc4730950b880fda3ef6f59ac82af0b268674d272aa61c69']
+
+builddependencies = [
+    ('CMake', '3.13.3', '', ('GCCcore', '8.2.0')),
+]
+dependencies = [
+    ('attr', '2.4.47', '', ('GCCcore', '8.2.0')),
+    ('bzip2', '1.0.6', '', ('GCCcore', '8.2.0')),
+    ('dtcmp', '1.1.0'),
+    ('libarchive', '3.4.0', '', ('GCCcore', '8.2.0')),
```

Comment:
specifying the subtoolchain isn't necessary, is it?

Reply:
Indeed, the hardcoded toolchains should be removed...